src/libraries/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/EncryptedReference.cs

```diff
@@ -30,6 +31,8 @@ protected EncryptedReference(string uri, TransformChain transformChain)
         public string Uri
         {
             get { return _uri; }
+
+            [MemberNotNull(nameof(_uri))]
```

Comment:
nit: that seems like rather weird annotation to have on the setter, I'd recommend avoiding it (feel free to ignore this assuming it doesn't flow to the ref)

Reply:
I'm pretty sure this was the result of other PR feedback. I've lost the history because of all the rebases required because of the age of this PR.